### ISSUE TITLE
Design System Staging Review - Converted v3 pagination component

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#66127]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** I'm not seeing the `aria-current="page"` on the current page in the pagination, eg. on the ["With Middle" story in Storybook](https://design.va.gov/storybook/?path=/docs/uswds-va-pagination--default#with-middle), the link for 10 is coded as:
   ```
   <li class="usa-pagination__item usa-pagination__page-no">
      <a href="javascript:void(0)" class="usa-pagination__button usa-current">10</a>
   </li>
   ```
   Per @rsmithadhoc's comments in the touchpoint meeting it sounds like this was already on your radar, just want to make sure it gets added.
- **Must:** The `nav` element in the component should have an `aria-label="pagination"` or similar so that screen reader users can easily distinguish it from other `nav` elements. I also noticed that in Ryan's review notes, so hopefully that can be addressed as well.
- **Should:** Per the discussion at the meeting, some guidance disallowing more than one pagination component per page would be helpful --- both to keep the page straightforward and understandable, and to avoid confusion from having multiple `nav` elements with `aria-label="pagination"`.

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/va.gov-team/66127?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc
